app/src/util/form-schema/manual-input-hierarchy.json

```diff
@@ -3593,31 +3806,37 @@
                 "required": false
               },
               {
-                "id": "incineration-waste-inboundary-waste-composition",
+                "id": "waste-composition",
                 "type": "percentage-breakdown",
                 "total-required": 100,
                 "subtypes": [
-                  "waste-composition-municipal-solid-waste",
-                  "waste-composition-industrial-solid-waste",
-                  "waste-composition-hazardous-waste",
                   "waste-composition-clinical-waste",
+                  "waste-composition-hazardous-waste",
+                  "waste-composition-industrial-solid-waste",
+                  "waste-composition-municipal-solid-waste",
                   "waste-composition-sewage-sludge"
                 ]
               },
               {
-                "id": "incineration-waste-inboundary-technology",
+                "id": "technology",
                 "options": [
                   "technology-continuous-incineration",
                   "technology-semi-continuous-incineration",
                   "technology-batch-type-incineration"
                 ]
               },
               {
-                "id": "incineration-waste-inboundary-boiler-type",
-                "options": ["boiler-type-stoker", "boiler-type-fluidised-bed"]
+                "id": "boiler-type",
```

Comment:
Sorry, I had to revert the PR with the activity input ID normalization as Evan said it would introduce too many breaking changes too late in the process.
So let's revert this to the old state (and the translation files as well) and use a name mapping in the formula to get the value depending on which activity is used.

Annoying, but nothing to be done about that for now...

Reply:
@lemilonkh  so for the changes I made. I made them to only the methodologies I worked with.  I also cross check the translation as that the only other place it was being used

Reply:
Yes, however changing only these and not the other ones we introduce breaking changes to our data model all the time. So let's do them all at once in a separate PR if possible.